internal/pkg/service/appsproxy/dataapps/api/provider_oidc.go

```diff
@@ -0,0 +1,10 @@
+package api
+
+type OIDCProvider struct {
+	BaseProvider
```

Comment:
We can have multiple types of providers, there is a task for `AzureAD` provider.